scaaml/utils.py

```diff
@@ -29,44 +29,44 @@ def pretty_hex(val):
     s = hex(int(val))
     s = s[2:]  # remove 0x
     if len(s) == 1:
-        s = '0' + s
+        s = "0" + s
     return s.upper()
 
 
-def bytelist_to_hex(lst, spacer=' '):
+def bytelist_to_hex(lst, spacer=" "):
     h = []
 
     for e in lst:
         h.append(pretty_hex(e))
     return spacer.join(h)
 
 
-def hex_display(lst, prefix="", color='green'):
+def hex_display(lst, prefix="", color="green"):
     "display a list of int as colored hex"
     h = []
     if len(prefix):
-        prefix += '\t'
+        prefix += "\t"
     for e in lst:
         h.append(pretty_hex(e))
     cprint(prefix + " ".join(h), color)
 
 
 def get_model_stub(attack_point, attack_byte, config):
-    return '%s-%s-%s-v%s-ap_%s-byte_%s-len_%s' % (
-        config['device'], config['algorithm'], config['model'],
-        config['version'], attack_point, attack_byte, config['max_trace_len'])
+    return "%s-%s-%s-v%s-ap_%s-byte_%s-len_%s" % (
```

Comment:
Unless this code requires to run with python 3.5, this should be converted into an f-string